airflow/www/static/js/dag.js

```diff
@@ -54,6 +54,9 @@ let executionDate = '';
 let subdagId = '';
 let dagRunId = '';
 let mapIndex;
+let runId;
```

Comment:
Does `dagRunId` not serve the same purpose?

Reply:
Yes, updated.